client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -205,6 +205,16 @@ export default function NotificationSettings( {
 		>
 			<div className="notification-settings__sub-title">{ getSiteCountText( sites ) }</div>
 
+			{ isBulkUpdate ? (
+				<div className="notification-settings__bulk-edit-warning">
+					<div className="notification-settings__bulk-edit-warning-block">
+						<div className="notification-settings__bulk-edit-warning-content">
+							{ translate( 'Settings for selected sites will be overwritten.' ) }
+						</div>
+					</div>
+				</div>
```

Comment:
I think we can remove the nested div and have only one div

```suggestion
				<div className="notification-settings__bulk-edit-warning">
					{ translate( 'Settings for selected sites will be overwritten.' ) }
				</div>
```